src/components/GameController.tsx

```diff
@@ -6,7 +6,23 @@ import WaitingStep from './WaitingStep';
 import SelectPlayStep from './SelectPlayStep';
 import * as playerAStates from '../game-engine/application-states/PlayerA';
 
-export default class GameController extends PureComponent {
+interface Props {
+  applicationState: playerAStates.PlayerAState;
+  chooseAPlay: any; // TODO: what should this be?
```

Comment:
Not sure what `chooseAPlay` is meant to do.

Reply:
That should be the function that triggers the action.